### DISCUSSION TITLE
Downloading the file..

### DISCUSSION THREAD

**Discussion by LoTech763** [fredemmott/StreamDeck-Discord#42] [Q&A] [answered]
asks me for an app to open up the file and then automatically opens the app in internet explorer. then asks constantly if I want to open it and save it. it's saved in my downloads but won't let me open it and install the plugin. it's just sitting in my downloads file. any suggestions as to what I should do to get the plug-in installed?

**Comment by fredemmott** [comment, marked as answer]
- Make sure you have the latest streamdeck software
- If there are no updates for the streamdeck software available, or installing them doesn't help, contact Elgato support.

This problem is completely out of control of the plugin

**Reply by LoTech763** [reply]
copy that, I just want to confirm when I download it asks me to open it with internet explorer, note pad, or some other app in the Microsoft Store. I open the file with Internet Explorer and it doesn't do anything just copies the file to my downloads file in my File Explorer. When I click the file it opens up I.E. downloads window and asks me to save the file and open it. when I click to open a glitched-out window shows up and then closes and it's just a repeating cycle at that point. I checked out my stream deck and I am running the latest version. Is there another way to open the file instead of through I.E. because if I open it through notepad it just gives me the coding?

**Reply by fredemmott** [reply]
Assuming you have a `.streamDeckPlugin` file downloaded, it should directly open in the streamdeck software, not internet explorer
